cg/store/api/status.py

```diff
@@ -212,6 +212,10 @@ def set_case_action(self, action: Literal[CASE_ACTIONS], case_id: str) -> None:
         case_obj.action = action
         self.commit()
 
+    def set_sample_comment(self, sample: models.Sample, comment: str) -> None:
```

Comment:
Refactor
 models.Class to just Class whenever there is a chance
```suggestion
    def add_sample_comment(self, sample:Sample, comment: str) -> None:
```